backend/services/season_service.go

```diff
@@ -0,0 +1,27 @@
+package services
+
+import (
+	"errors"
+	"gorm.io/gorm"
+	database "mmr/backend/db"
+	"mmr/backend/db/models"
+	"mmr/backend/db/repos"
+)
+
+type SeasonService struct{}
+
+func (ss SeasonService) CurrentSeasonID() uint {
+	seasonRepo := repos.NewSeasonRepository(database.DB)
+	season, err := seasonRepo.CurrentSeason()
+	if err != nil {
+		if errors.Is(err, gorm.ErrRecordNotFound) {
+			newSeason, newErr := seasonRepo.CreateSeason(&models.Season{})
```

Comment:
Kan ikke helt finde ud af om den her sideeffekt bør være der... Vi har jo ikke brug for den lige nu fordi du opretter season migreringen. Og når vi laver noget mere til at håndtere seasons, så er det nok heller ikke denne måde vi vil oprette på

Reply:
Sandt 👍